src/Components/Blazor/Build/src/targets/All.props

```diff
@@ -1,8 +1,12 @@
 <Project>
   <Import Project="Blazor.MonoRuntime.props" />
 
+  <Import Project="StaticWebAssets.props" />
+
   <PropertyGroup>
-    <DefaultWebContentItemExcludes>$(DefaultWebContentItemExcludes);wwwroot\**</DefaultWebContentItemExcludes>
```

Comment:
I don't really know why this was here before. Did you remove it just because it serves no purpose? Or some other reason?

Reply:
It was getting in the way of discovering assets inside the wwwroot folder.